### ISSUE TITLE
Each host app, should use its own conversation in overlay

### ISSUE THREAD

**Issue by VolhaBazhkova** [epam/ai-dial-chat#569]
1. Open Overlay1 connected to site1 (example, {SUB-PROJECT-1} + overlay)
2. Send any message in chat
3. In the same browser on new tab open Overlay2 connected to site2 (example, landing page +overlay)
4. Note that the response in Overlay1 appears also in Overlay2 -> **issue**

**Expected result:** each host app, should use its own conversation in overlay

**Note:** similar ticket was fixed in #545 Please, re-test it after the new implementation

**Comment by mikitabut** [comment]
Agreed to have selected conversation specific to overlay host domain
